project/ws/app/src/lib/routes/user-profile/components/request-dialog/request-dialog.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="main_container mt-4 flex-row">
+    <div class="">
+        <div class="right_inner_container profile-box">
+            <div class="flex">
+                <h2 class="mat-h1 margin-remove inline-block">
+                    Request to add designation
```

Comment:
I think designation should be Designation

Reply:
It is fine as same coming on the registration page